### ISSUE TITLE
DOM7011: The code on this page disabled back and forward caching

### ISSUE THREAD

**Issue by angeloocana** [gsklee/ngStorage#183]
Only in IE 11, i'm getting the errors below:

DOM7011: The code on this page disabled back and forward caching. For more information, see: http://go.microsoft.com/fwlink/?LinkID=291337
staffgeist.azurewebsites.net

HTML1300: Navigation occurred.
staffgeist.azurewebsites.net

can anyone help me?

**Comment by samuelneff** [comment]
Those are not errors, just informational. Happens in Angular when using HTML5 mode since our Angular code is handling back/forward instead of the browser.

**Comment by SONALIKOKARE** [comment]
Yes it's just an infomational.But then how to run our angular js code on IE browser.

**Comment by Mozar10** [comment]
Has anything been worked out for this error?

**Comment by biswajitpanday** [comment]
Is there any update..?

**Comment by melban** [comment]
I'm having the same issue with my Meteor app. Works fine in Chrome but IE and Safari don't render.

**Comment by Allen-Porter** [comment]
Despite it being an informational, a practical consequence appears to be that IE debugger DOM output is disabled.
